### ISSUE TITLE
'ckan install -c' installs latest version in repository instead of the ckan supplied

### ISSUE THREAD

**Issue by techman83** [KSP-CKAN/CKAN#1356]
Highlighted during metadata testing.

```
+ echo Running ckan install -c TriggerAu-Flags/TriggerAu-Flags-2.7.1.0.ckan
Running ckan install -c TriggerAu-Flags/TriggerAu-Flags-2.7.1.0.ckan
+ mono --debug ckan.exe install -c TriggerAu-Flags/TriggerAu-Flags-2.7.1.0.ckan --headless
About to install...

 * TriggerAu-Flags 2.7.2.0
```

**Comment by HebaruSan** [comment]
This makes pull request validation fairly useless when updating old versions in CKAN-meta; see KSP-CKAN/CKAN-meta#1310, in which metadata for older versions of three mods was updated and then tested by installing the latest versions instead.

**Comment by HebaruSan** [comment]
Fixed in #2211 (GitHub lost the linkage due to rebasing).
